src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,27 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        try {
+            List<String> strings = Files.readAllLines(file.toPath());
+            System.out.println(strings);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
 
+        try {
+            Files.write(file.toPath(), toFileName.getBytes(), StandardOpenOption.APPEND);
```

Comment:
The current implementation writes to the same file (`fromFileName`) instead of `toFileName`. You should create a new `File` object for `toFileName` and write the output to it.